x-pack/test/functional/services/uptime/settings.ts

```diff
@@ -25,24 +26,24 @@ export function UptimeSettingsProvider({ getService }: FtrProviderContext) {
       await changeInputField(text, 'heartbeat-indices-input-loaded');
     },
     changeErrorThresholdInput: async (text: string) => {
-      await changeInputField(text, 'error-state-threshold-input-loaded');
+      await changeInputField(text, 'expiration-threshold-input-loaded');
     },
     changeWarningThresholdInput: async (text: string) => {
-      await changeInputField(text, 'warning-state-threshold-input-loaded');
+      await changeInputField(text, 'age-threshold-input-loaded');
     },
-    loadFields: async () => {
+    loadFields: async (): Promise<DynamicSettings> => {
       const indInput = await testSubjects.find('heartbeat-indices-input-loaded', 5000);
-      const errorInput = await testSubjects.find('error-state-threshold-input-loaded', 5000);
-      const warningInput = await testSubjects.find('warning-state-threshold-input-loaded', 5000);
+      const errorInput = await testSubjects.find('expiration-threshold-input-loaded', 5000);
+      const warningInput = await testSubjects.find('age-threshold-input-loaded', 5000);
```

Comment:
var name rename plz :)

Reply:
Renamed in https://github.com/elastic/kibana/pull/64111/commits/f7678b685567f28ccb34c78f6010710b17c1b4b7